webrender/src/renderer.rs

```diff
@@ -2215,7 +2217,7 @@ pub struct RendererOptions {
     pub enable_batcher: bool,
     pub render_target_debug: bool,
     pub max_texture_size: Option<u32>,
-    pub workers: Option<Arc<Mutex<ThreadPool>>>,
+    pub workers: Option<Arc<Mutex<rayon::ThreadPool>>>,
```

Comment:
Do you need the mutex now? It seems the `spawn_async` function takes an immutable `ThreadPool`, so I don't see why off-hand.